.github/workflows/build-test.yml

```diff
@@ -1,6 +1,14 @@
-name: CMake
+name: build-tests
```

Comment:
In order to keep with the yml file name, the name value could be renamed to: build-test